spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreQueryOptions.java

```diff
@@ -33,10 +35,13 @@
 
 	private Sort sort;
 
-	public DatastoreQueryOptions(Integer limit, Integer offset, Sort sort) {
+	private Cursor cursor;
+
+	public DatastoreQueryOptions(Integer limit, Integer offset, Sort sort, Cursor cursor) {
```

Comment:
Explain in javadoc what can and cannot be null.
Generally, nullable params should be set through setters, not constructor.
Consider a builder.

Reply:
added a builder